extension.ts

```diff
@@ -39,21 +39,23 @@ export class WordCounter {
         }
 
         let doc = editor.document;
+        let sel = editor.selection;
 
         // Only update status if an MD file
         if (doc.languageId === "markdown") {
             let wordCount = this._getWordCount(doc);
+            let selectedWordCount = this._getWordCount(doc, sel);
 
             // Update the status bar
-            this._statusBarItem.text = wordCount !== 1 ? `$(pencil) ${wordCount} Words` : '$(pencil) 1 Word';
+            this._statusBarItem.text = wordCount !== 1 ? `$(pencil) ${wordCount} Words | ${selectedWordCount} selected` : '$(pencil) 1 Word';

```

Comment:
Could this be omitted if the selectedWordCount is zero?

Reply:
That is how this works, if `selectedWordCount` is 0 (i.e. no selection is active or blank space is selected) the total `wordCount` is displayed instead of `selectedWordCount`.